### PR TITLE
Make audio notifications work

### PR DESCRIPTION
**Pull request by dbkr**
https://github.com/matrix-org/matrix-react-sdk/pull/191 from @ajohnson23, with the addition of only playing audio if the sound tweak is set on the notification.

Requires https://github.com/vector-im/vector-web/pull/1100 to work (but safe without).

### CODE REVIEW

**Comment by dbkr** [comment]
Fixes https://github.com/vector-im/vector-web/issues/167

File: src/Notifier.js [hunk 1]
```diff
@@ -83,15 +83,26 @@ var Notifier = {
...
+
+        var audioClip;
+        if (actions.tweaks.sound) {
+            audioClip = playAudio();
+        }
 
         global.setTimeout(function() {
             notification.close();
+            if (audioClip) {
+                audioClip.pause();
```
Comment (richvdh) [inline src/Notifier.js]: Why is this necessary?

Comment (dbkr) [inline src/Notifier.js]: Actually this probably isn't necessary, and would cause the notification sound to stop mid-way if a second notif came in and then a previous one hit the timeout.

**Comment by dbkr** [comment]
ptal

**Comment by richvdh** [comment]
Does it work? Lgtm if so

**Comment by dbkr** [comment]
Yep, tested & it works.
